packages/file-search/src/browser/quick-file-open.ts

```diff
@@ -153,7 +153,7 @@ export class QuickFileOpenService implements QuickOpenModel, QuickOpenHandler {
         const locations = [...this.navigationLocationService.locations()].reverse();
         for (const location of locations) {
             const uriString = location.uri.toString();
-            if (!alreadyCollected.has(uriString) && fuzzy.test(lookFor, uriString)) {
+            if (!alreadyCollected.has(uriString) && fuzzy.test(lookFor, uriString) && !uriString.startsWith('diff')) {
```

Comment:
Does it make sense to register anything but `file` URIs?

```ts
if (location.uri.scheme === 'file' && !alreadyCollected.has(uriString) && fuzzy.test(lookFor, uriString))
```

Reply:
I like your implementation better :) I updated the PR.